core/include/traccc/finding/finding_algorithm.ipp

```diff
@@ -20,28 +20,55 @@ template <typename stepper_t, typename navigator_t>
 track_candidate_container_types::host
 finding_algorithm<stepper_t, navigator_t>::operator()(
     const detector_type& det,
-    const measurement_container_types::host& measurements,
+    const measurement_collection_types::host& measurements,
     const bound_track_parameters_collection_types::host& seeds) const {
 
     track_candidate_container_types::host output_candidates;
 
-    /**********************
-     * Create module map
-     **********************/
-
-    std::vector<std::pair<geometry_id, unsigned int>> module_map;
-    const int n_modules = measurements.size();
+    /*****************************************************************
+     * Measurement Operations
+     *****************************************************************/
+
+    // Copy the measurements
+    measurement_collection_types::host sorted_measurements;
+    std::copy(measurements.begin(), measurements.end(),
+              std::back_inserter(sorted_measurements));
```

Comment:
I think this is perfectly valid, but to be honest I would've just gone with the much simpler:

```c++
measurement_collection_types::host sorted_measurements = 
    measurements;
```

:wink:

Reply:
Makes sense